planets/eve/explodium sea/barometerScan.cfg

```diff
@@ -2,6 +2,14 @@
 {
 	%RESULTS
 	{
-		//LAST_ENTRY
+                // Science Data is available in different situations for each
+                // planetary body. Please use this framework as a guide to the
+                // situational science that is available for this body. You may
+                // copy-paste and remove '//' to add your entry. Please add your
+                // entry just before '//LAST_ENTRY' or with similar entries.
+                //EveSrfSplashedExplodiumSea = Your science report text here.
```

Comment:
You can also land in water so you can add "EveSrfLandedExplodiumSea = Your science report text here."